src/cardStyles/stars.css

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.stars__star {
+  margin-left: 4px;
+  list-style: none;
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars .stars__star:nth-child(1) {
+  margin-left: 0;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
```

Comment:
you should add all the possible modifiers, just like you did in `stars` task

Reply:
All modifiers  gave been add .